library/src/hcc_detail/hipsparse.cpp

```diff
@@ -13714,7 +13744,6 @@ hipsparseStatus_t hipsparseSpGEMM_copy(hipsparseHandle_t      handle,
     return status;
 }
 
-HIPSPARSE_EXPORT
```

Comment:
why do we remove this keyword?

Reply:
Its not needed in the .cpp file. Only required in the .h file. I removed the few instances of it in this file as I assume they were probably there from accidentally copying the signature from the .h file to the .cpp file